MAINTAINERS.yml

```diff
@@ -0,0 +1,1052 @@
+        - samples/subsys/power/
+        - subsys/power/
+    labels:
+        - "area: Power Management"
+
+"RISCV arch":
+    # Provisional entry, to be corrected
+    status: maintained
+    maintainers:
+        - nategraff-sifive
+    collaborators:
+        - mgielda
+    files:
+        - arch/riscv/
+        - include/arch/riscv/
+    labels:
+        - "area: RISCV"
+
+Sanitycheck:
+    # Provisional entry, to be corrected
+    status: maintained
+    maintainers:
+        - nashif
+    files:
+        - scripts/sanitycheck
+        - scripts/sanity_chk/
+    labels:
+        - "area: Sanitycheck"
+
+Settings:
+    # Provisional entry, to be corrected
+    status: maintained
+    maintainers:
+        - nvlsianpu
+    files:
+        - include/settings/
+        - subsys/settings/
+        - tests/subsys/settings/
+    labels:
+        - "area: Settings"
+
+Shell:
+    # Provisional entry, to be corrected
+    status: maintained
+    maintainers:
+        - jakub-uC
+    files:
+        - include/shell/
+        - samples/subsys/shell/
+        - subsys/shell/
+        - tests/shell/
+        - tests/subsys/shell/
+    labels:
+        - "area: Shell"
+
+Shields:
+    status: maintained
+    maintainers:
+        - erwango
+    collaborators:
+        - avisconti
+        - jfischer-phytec-iot
+    files:
+        - boards/shields/
+    labels:
+        - "area: Shields"
+
+STM32:
+    status: maintained
+    maintainers:
+        - erwango
+    files:
+        - boards/arm/nucleo_*/
+        - boards/arm/stm32*_disco/
+        - boards/arm/stm32*_eval/
+        - drivers/*/*stm32*/
+        - drivers/*/*/*stm32*
+        - dts/arm/st/
+        - dts/bindings/*/*stm32*
+        - soc/arm/st_stm32/
+    labels:
+        - "platform: STM32"
+    description: >-
+        STM32 SOCs, dts files and related drivers. ST nucleo, disco and eval
+        boards.
+
+"ZTest and unit test infrastructure":
+    # TODO @nashif is this what line 59 of the spreadsheet refers to?
+    # Provisional entry, to be corrected
+    status: maintained
+    maintainers:
+        - nashif
+    files:
+        - subsys/testsuite/
+        - tests/ztest/
+        - tests/unit/util/
+    labels:
+        - "area: Debugging"
+
+USB:
+    # Provisional entry, to be corrected
+    status: maintained
+    maintainers:
+        - finikorg
+    collaborators:
+        - jfischer-phytec-iot
+    files:
+        - drivers/usb/
+        - dts/bindings/usb/
+        - include/*/usb/
+        - include/usb/
+        - samples/subsys/usb/
+        - subsys/usb/
+        - tests/subsys/usb/
+    labels:
+        - "area: USB"
+
+Userspace:
+    # Provisional entry, to be corrected
+    status: maintained
+    maintainers:
+        - andrewboie
+    files:
+        - doc/reference/usermode/kernelobjects.rst
+        - include/app_memory/
+        - include/linker/app_smem*.ld
+    labels:
+        - "area: Userspace"
+
+West:
+    # Provisional entry, to be corrected
```

Comment:
confirmed; I am the maintainer. @carlescufi as collaborator please

Reply:
@carlescufi : Procedural check: Can you confirm that you want to be listed as collaborator for west?

Reply:
Yes please